cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,20 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+  it('should ', () => {
+    // Відвідати сторінку "Sign in"
```

Comment:
```suggestion
```
remove extra comments, why did you add them?

Reply:
Fixed